_posts/elements/2015-04-09-buttons.md

```diff
@@ -0,0 +1,97 @@
+---
+layout:       element
+title:        Buttons
+date:         2015-04-09 00:00:00
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- headings
+
+slug:         buttons
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/_src/static/sass/components/_buttons.scss"
+
+description:  Examples of the different types and variants of edX buttons.
+
+info: Buttons are great. Mkay.
+---
+
+<section class="example-set">
```

Comment:
This is a nice convention. Can we sync up other PL posts with these ``<section class="example-set">`` containers if they are not already (in a separate PR from this work)?